api/server/runner.go

```diff
@@ -137,6 +146,33 @@ func (s *Server) handleRequest(c *gin.Context, enqueue models.Enqueue) {
 	c.JSON(http.StatusNotFound, simpleError(models.ErrRunnerRouteNotFound))
 }
 
+func processJwt(signingKey string, c *gin.Context) error {
```

Comment:
since we are (probably) going to re use this in `server.go` when implementing JWT for base API calls, I think we should refactor this (and jwt dependencies) out of here, and into a separate package where they can be referenced from `server` and `commands`.